clusters/app.ci/release-controller/admin_deploy-ocp-publish-art.yaml

```diff
@@ -164,6 +164,55 @@ subjects:
 # art_publish_permissions.py added the following resource
 #---------------------------------------------------------------------------------
 
+# Allow ART to patch releases (accept/reject) in the ocp namespace
+apiVersion: authorization.openshift.io/v1
+kind: ClusterRole
+metadata:
+  name: art-publish-cluster
```

Comment:
This name seems weird to me...
Perhaps something like: `art-publish-release-admin`?

Reply:
Agreed that's better :+1: